.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+db.sqlite3
```

Comment:
ok but common practice is to use pregenerated gitignore, since you don't always know all files you should inore.